Neos.Neos/Classes/FrontendRouting/FrontendNodeRoutePartHandlerInterface.php

```diff
@@ -20,8 +20,6 @@
  * Marker interface which can be used to replace the currently used FrontendNodeRoutePartHandler,
  * to e.g. use the one with localization support.
  *
- * TODO CORE MIGRATION
- *
```

Comment:
what _can_ we migrate here anyway? The class name changed but extending it will render any custom code useless probably